src/top_down_iterator.jl

```diff
@@ -33,15 +33,13 @@ function priority_function(
 end
 
 """
-    function derivation_heuristic(::TopDownIterator)
+    function derivation_heuristic(::TopDownIterator, indices::Vector{Int})
 
 Returns a sorted sublist of the `indices`, based on which rules are most promising to fill a hole.
 By default, this is the identity function.
 """
-function derivation_heuristic(::TopDownIterator)
-    return function (indices)
-        return indices;
-    end
+function derivation_heuristic(::TopDownIterator, indices::Vector{Int})
+    return indices;
```

Comment:
You probably forgot to delete the simicolon here 👍

Reply:
The semicolon is optional here. Do we apply the convention to omit optional semicolons? There are quite some of them in other places too.

Reply:
We've just added `JuliaFormatter` support to `HerbCore`, and I'm going to add it to all of the repos (soon ™️!). When we run it, it will strip all of the extraneous trailing semicolons among other stylistic changes.

Don't worry about it for now.

Reply:
Cool!

Reply:
I can't wait for the JuliaFormatter 👍